src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
It seems to me that no matter what it _shouldn't_ want to give you a null, cause it should only be null if there is no "reference transform" and every vessel (controlled or not) should have a reference transform.  I'd have to dig through ILSpy to verify, but it very well could be that a ship with no "control" when it loads doesn't get the "reference transform" set (which might also be an interesting interaction with remote tech).